scheduler/strategy/binpack.go

```diff
@@ -0,0 +1,27 @@
+package strategy
+
+import (
+	"sort"
+
+	"github.com/docker/swarm/cluster"
+	"github.com/samalba/dockerclient"
+)
+
+type BinpackPlacementStrategy struct {
+}
+
+func (p *BinpackPlacementStrategy) Initialize() error {
+	return nil
+}
+
+func (p *BinpackPlacementStrategy) PlaceContainer(config *dockerclient.ContainerConfig, nodes []cluster.Node) (cluster.Node, error) {
+	weightedNodes, err := weightNodes(config, nodes)
+	if err != nil {
+		return nil, err
+	}
+
+	// sort by highest weight
```

Comment:
it's the same comment as in your PR https://github.com/docker/swarm/pull/398/files

If it's not clear, even for you, we have an issue :D